angular-chart.js

```diff
@@ -193,13 +202,16 @@
               getData(scope.labels, scope.data, scope.colours);
             var options = angular.extend({}, ChartJs.getOptions(type), scope.options);
             chart = new ChartJs.Chart(ctx)[type](data, options);
-            scope.$emit('create', chart);
+            scope.$emit('chart-created', chart);
```

Comment:
This would be a breaking change, it will be in the next version that will add support for Chart.js 2.0. Also in general it's better to keep PRs focused on fixing one issue at a time. Pls remove this change.